packages/content/README.md

```diff
@@ -42,3 +42,30 @@ if (data) {
   });
 }
 ```
+
+## Content Header
```

Comment:
Would make the below changes in https://github.com/thoughtindustries/ti-docs/pull/18 as well.